index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Valentine Jeremiah</h1>
+      <h3 class="sub-title">The Best Microverse pair programers</h3>
+      <figure id="img-div">
+        <img src="./images/Picture1-1.png" alt="" id="image">
+        <h3 id="img-caption">This is Jeremiah and Valentine</h3>
+      </figure>
+      <div id="tribute-info">Jeremiah from Ghana and Valentine from Nigeria were paired in microverse admission trial to complete few coding challenges and a project. 
+        <br>
+        <br>
+        We both Choose to do the tribute project after completing 2 challenges from hackerank. It have been fun and a great learning proccess for both of us. We enjoy working with each other and will definatly will love to work together in future projects. 
```

Comment:
Please wrap the sentence in a paragraph 'p' tag. This is for better code readability

Reply:
- Good job being polite while giving your review 👏, and for also stating the reason for your suggested change 💯 💯
- Please, I'd suggest you try using emojis while giving your reviews, this will make you sound more friendly 😄 😄